src/net/sourceforge/zmanim/hebrewcalendar/JewishDate.java

```diff
@@ -141,6 +141,19 @@
 	 */
 	public static final int ADAR_II = 13;
 
+	public static final int MONTH_JANUARY = Calendar.JANUARY + 1;

```

Comment:
I think it could get confusing to folks who know about (and are used to) the Calendar constants to have this alternate set of constants. Especially when these are not interchangeable with the Calendar constants.

Perhaps it'd be better to think of how we could make the API easier to interact with the Calendar constants?

Reply:
MONTH_ consts are internal.

Reply:
fixed in https://github.com/KosherJava/zmanim/pull/122